neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -0,0 +1,185 @@
+            [OpCode.PUSHBYTES3] = 120,
+            [OpCode.PUSHBYTES4] = 120,
+            [OpCode.PUSHBYTES5] = 120,
+            [OpCode.PUSHBYTES6] = 120,
+            [OpCode.PUSHBYTES7] = 120,
+            [OpCode.PUSHBYTES8] = 120,
+            [OpCode.PUSHBYTES9] = 120,
+            [OpCode.PUSHBYTES10] = 120,
+            [OpCode.PUSHBYTES11] = 120,
+            [OpCode.PUSHBYTES12] = 120,
+            [OpCode.PUSHBYTES13] = 120,
+            [OpCode.PUSHBYTES14] = 120,
+            [OpCode.PUSHBYTES15] = 120,
+            [OpCode.PUSHBYTES16] = 120,
+            [OpCode.PUSHBYTES17] = 120,
+            [OpCode.PUSHBYTES18] = 120,
+            [OpCode.PUSHBYTES19] = 120,
+            [OpCode.PUSHBYTES20] = 120,
+            [OpCode.PUSHBYTES21] = 120,
+            [OpCode.PUSHBYTES22] = 120,
+            [OpCode.PUSHBYTES23] = 120,
+            [OpCode.PUSHBYTES24] = 120,
+            [OpCode.PUSHBYTES25] = 120,
+            [OpCode.PUSHBYTES26] = 120,
+            [OpCode.PUSHBYTES27] = 120,
+            [OpCode.PUSHBYTES28] = 120,
+            [OpCode.PUSHBYTES29] = 120,
+            [OpCode.PUSHBYTES30] = 120,
+            [OpCode.PUSHBYTES31] = 120,
+            [OpCode.PUSHBYTES32] = 120,
+            [OpCode.PUSHBYTES33] = 120,
+            [OpCode.PUSHBYTES34] = 120,
+            [OpCode.PUSHBYTES35] = 120,
+            [OpCode.PUSHBYTES36] = 120,
+            [OpCode.PUSHBYTES37] = 120,
+            [OpCode.PUSHBYTES38] = 120,
+            [OpCode.PUSHBYTES39] = 120,
+            [OpCode.PUSHBYTES40] = 120,
+            [OpCode.PUSHBYTES41] = 120,
+            [OpCode.PUSHBYTES42] = 120,
+            [OpCode.PUSHBYTES43] = 120,
+            [OpCode.PUSHBYTES44] = 120,
+            [OpCode.PUSHBYTES45] = 120,
+            [OpCode.PUSHBYTES46] = 120,
+            [OpCode.PUSHBYTES47] = 120,
+            [OpCode.PUSHBYTES48] = 120,
+            [OpCode.PUSHBYTES49] = 120,
+            [OpCode.PUSHBYTES50] = 120,
+            [OpCode.PUSHBYTES51] = 120,
+            [OpCode.PUSHBYTES52] = 120,
+            [OpCode.PUSHBYTES53] = 120,
+            [OpCode.PUSHBYTES54] = 120,
+            [OpCode.PUSHBYTES55] = 120,
+            [OpCode.PUSHBYTES56] = 120,
+            [OpCode.PUSHBYTES57] = 120,
+            [OpCode.PUSHBYTES58] = 120,
+            [OpCode.PUSHBYTES59] = 120,
+            [OpCode.PUSHBYTES60] = 120,
+            [OpCode.PUSHBYTES61] = 120,
+            [OpCode.PUSHBYTES62] = 120,
+            [OpCode.PUSHBYTES63] = 120,
+            [OpCode.PUSHBYTES64] = 120,
+            [OpCode.PUSHBYTES65] = 120,
+            [OpCode.PUSHBYTES66] = 120,
+            [OpCode.PUSHBYTES67] = 120,
+            [OpCode.PUSHBYTES68] = 120,
+            [OpCode.PUSHBYTES69] = 120,
+            [OpCode.PUSHBYTES70] = 120,
+            [OpCode.PUSHBYTES71] = 120,
+            [OpCode.PUSHBYTES72] = 120,
+            [OpCode.PUSHBYTES73] = 120,
+            [OpCode.PUSHBYTES74] = 120,
+            [OpCode.PUSHBYTES75] = 120,
```

Comment:
The same cost for 75 bytes as 1?

Reply:
Yes. Their prices are very close, so they are set to the same.